presto-docs/src/main/sphinx/optimizer/cost-in-explain.rst

```diff
@@ -2,14 +2,14 @@
 Cost in EXPLAIN
```

Comment:
It's a good point. I do see however a benefit of explaining how did we end up having a cost and what it means in relation to stats. But maybe we we can split and cross-link the doc pages.